Tasks/ContainerStructureTestV0/testrunner.ts

```diff
@@ -18,11 +19,19 @@ export class TestRunner {
                     throw new Error("Unable to get runner download path");
```

Comment:
nit: This error message is misleading and should instead be "Not supported OS type" since we aren't doing any fetch of the download path but instead relying on a mapping of OS type to download url

Reply:
I am doing the OS check in the start of the script itself, The code won't reach this method if os is windows.

Reply:
I have changed the error message.